robot/basestation/static/js/pages/arm.js

```diff
@@ -202,7 +202,7 @@ function gameLoop () {
       toggleToManual()
       $('#click_btn_motor1_ccw > button').css(
         'background-color',
-        'rgb(255, 0, 0)'
+        'rgb(63, 97, 140)'
```

Comment:
This smells like a magic number, although technically it would be more of a magic tuple.
Perhaps we can declare it as a variable (string) at the top for this reason?